src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -383,7 +383,15 @@ namespace AirLoopHVACDOAS {
 
                 thisSplitter.name = Util::makeUPPER(thisObjectName);
                 thisSplitter.InletNodeName = Util::makeUPPER(fields.at("inlet_node_name").get<std::string>());
-                thisSplitter.InletNodeNum = Util::FindItemInList(thisSplitter.InletNodeName, state.dataLoopNodes->NodeID);
+                thisSplitter.InletNodeNum = NodeInputManager::GetOnlySingleNode(state,
```

Comment:
The correct function is used to get an air node number.

Reply:
@lgu1234 This looks good. The same change is needed for the Mixer inlet nodes and the Splitter outlet nodes.

Reply:
@mjwitte Thanks. The same changes are made.